spec/requests/posts_controller_spec.rb

```diff
@@ -2411,6 +2411,9 @@
         },
       )
     end
+    let(:tag_only_revision) do
+      Fabricate(:post_revision, post: post, modifications: { "tags" => %w[tag1,tag2 tag2,tag3] })
```

Comment:
Let's add more test cases here - what happens when tags are not added in lexical order?